lib/country_select/tag_helper.rb

```diff
@@ -15,7 +15,8 @@ def country_option_tags
 
         if priority_countries_options.map(&:second).include?(option_tags_options[:selected])
           option_tags_options[:selected] = nil
-        end
+        end unless option_tags_options[:selected].kind_of?(Array)
+        option_tags_options[:selected].delete_if{|selected| priority_countries_options.map(&:second).include?(selected)} if option_tags_options[:selected].kind_of?(Array)
```

Comment:
Since you check again if `option_tags_options[:selected].kind_of?(Array)` maybe you should store that value in a variable _or_ just have two separate blocks for an array vs non-array `selected` option.